app/graphql/mutations/repository/remove_url_mapping_mutation.rb

```diff
@@ -15,7 +15,7 @@ module Repository
       end
 
       resource!(lambda do |_root, arguments, _context|
-        ::Repository.first(slug: arguments[:repositoryId])
+        ::RepositoryCompound.first(slug: arguments[:repositoryId])
```

Comment:
I think you can remove the `::` now (It was only needed because we are also in a Module named `Repository`, so it didn't know which `Repository` we meant.